pkg/analyser/testdata/output.json

```diff
@@ -40,7 +40,12 @@
 		{
 			"res": {
 				"id": "AKIA5QYBVVD25KFXJHYJ",
-				"type": "aws_iam_access_key"
+				"type": "aws_iam_access_key",
+				"source": {
+					"source": "tfstate://terraform.tfstate",
+					"namespace": "module",
```

Comment:
What's the difference between `namespace` and `module` ?

Reply:
Namespace is a generic meaning term, for a terraform module IaC source it is bound to the notion of module, but it can be potentially bound to another notion for other IaC sources (Cloudformation, etc ...)